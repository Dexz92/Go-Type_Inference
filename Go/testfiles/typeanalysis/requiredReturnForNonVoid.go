func main() {
  var a int = help();
}

func help() int {
  
}
