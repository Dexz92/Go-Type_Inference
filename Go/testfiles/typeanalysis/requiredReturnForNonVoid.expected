Error at line 5: Function help() missing return statement
